example/App.js

```diff
@@ -32,7 +33,19 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text style={styles.titleText}>Emoji Mart Native</Text>
+        <View style={styles.titleContainer}>
+          <Text style={styles.titleText}>Emoji Mart Native</Text>
+          <Emoji emoji="department_store" size={16} />
```

Comment:
@ex7r3me I've had a little more think about this one, let's remove this `<Emoji/>` and replace `santa` with `department_store` above :+1:

Reply:
@pederjohnsen Alright, sorry for the delay, I had an intense week with multiple interviews which happily went good, I can get back on the sample and will make the changes ASAP.